frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -126,6 +137,11 @@ action.GitMachete.FastForwardParentToMatchSelectedBranchAction.GitMacheteContext
 action.GitMachete.FastForwardParentToMatchSelectedBranchAction.description=Fast forward parent to match selected branch
 
 
+action.GitMachete.OverrideForkPointAction.text=Git Machete: Override Fork Point Of Selected Branch
+action.GitMachete.OverrideForkPointAction.GitMacheteContextMenu.text=O_verride Fork Point Of Branch
```

Comment:
`Of Selected Branch`/`Of Branch` - intentional?

Reply:
![image](https://user-images.githubusercontent.com/19799111/88915010-cbbd5680-d263-11ea-93e8-dda9728e558d.png)

Git Log context menu :arrow_up: 

Our idea in the first place was to add `selected` for descriptions & "general" text (seen within `Find Action...`) but do not use it within context menu (IMHO it is redundant here and inconsistent with the git log, the image above). 

I can see that we crossed over this convention at some point (slide in/out, fast forward). I will fix them.

Reply:
issue created #461